benchmarks/argmax/one_workgroup_argmax_subgroup.glsl

```diff
@@ -0,0 +1,44 @@
+# version 450 core
+#extension GL_EXT_control_flow_attributes : enable
+#extension GL_KHR_shader_subgroup_arithmetic : enable
+#extension GL_KHR_shader_subgroup_ballot : enable
+
+layout(local_size_x = 16, local_size_y = 1, local_size_z = 1) in;
```

Comment:
This is smaller than the native subgroup size for desktop GPU. How does this perform if we increase this to `32` or `64`?

Reply:
![image](https://github.com/user-attachments/assets/0b2ad689-45d5-4060-a9f6-4efd556ca3f3)
![image](https://github.com/user-attachments/assets/06cb2632-155d-47fa-bc5c-41ee1f00efd8)

Reply:
I did not see any performance improvement